docs/content/testnets.rst

```diff
@@ -0,0 +1,13 @@
+Testnets
+========
+
+brief blurbs then links to docs about it
+
+Create
+------
+
+Join
```

Comment:
Different people want to join. 
- Full Node
- Validators (Full node ++)
- End-users (links to voyager binaries)